src/taipy/config/config.py

```diff
@@ -507,8 +507,38 @@ def configure_csv_data_node(
         Returns:
             DataNodeConfig^: The new CSV data node configuration.
         """
-        return cls.configure_data_node(id, DataNodeConfig._STORAGE_TYPE_VALUE_CSV, scope=scope,
-                                       default_path=default_path, has_header=has_header, **properties)
+        return cls.configure_data_node(
+            id,
+            DataNodeConfig._STORAGE_TYPE_VALUE_CSV,
+            scope=scope,
+            default_path=default_path,
+            has_header=has_header,
+            **properties,
+        )
+
+    @classmethod
+    def configure_json_data_node(
+        cls,
+        id: str,
+        default_path: str = None,
+        scope=DataNodeConfig._DEFAULT_SCOPE,
```

Comment:
We discussed the idea of asking the user to provide a Python class to represent the json object. We should add it in the signature. No?

Reply:
I think we can ask the user to provide a custom data class with a "from_dict" function which we can use and call to do the conversion :D

Reply:
@khaitruong922 if it's a list, we can turn it into a list of custom data class objects. What do you think?

Reply:
@toan-quach sounds good to me

Reply:
great! dont hesitate to let me know if you need any help :D I have worked on similar topics before so I think I can help you out if you need :D

Reply:
I was thinking of adding the `exposed_type` to configure json but it is also not in the configure csv and excel too.